src/Analyzer/Pass/Expression/FunctionCall/ArgumentUnpacking.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace PHPSA\Analyzer\Pass\Expression\FunctionCall;
+
+use PhpParser\Node\Expr\FuncCall;
+use PhpParser\Node\Name;
+use PHPSA\Compiler\Expression;
+use PHPSA\Context;
+use PHPSA\Definition\ClassMethod;
+
+class ArgumentUnpacking extends AbstractFunctionCallAnalyzer
+{
+
+    public function pass(FuncCall $funcCall, Context $context)
+    {
+        $functionName = $this->resolveFunctionName($funcCall, $context);
+        if ($functionName === "func_get_args") {
```

Comment:
In some cases $functionName can be null or boolean, on so dynamic context and cannot be resolved
Please add check that it not null like:

```php
if ($functionName && $functionName === 'func_get_args') {
```

And use '' instead of ""
"func_get_args" -> 'func_get_args'

Thanks